contracts/interchain-token-service/tests/utils/mod.rs

```diff
@@ -52,11 +55,13 @@ pub fn setup_env<'a>() -> (
 
     let (signers, gateway_client) = setup_gateway(&env, 0, 5);
     let gas_service_client = setup_gas_service(&env);
+
     let client = setup_its(&env, &gateway_client, &gas_service_client);
 
     (env, client, gateway_client, signers)
 }
 
+#[allow(dead_code)]
```

Comment:
need to fix our test organization so we don't need to add this stuff

Reply:
yeah, I will do it in a separate PR. created a ticket: https://axelarnetwork.atlassian.net/browse/AXE-6873